### ISSUE TITLE
Keras Implementation

### ISSUE THREAD

**Issue by jinyeom** [whitewolf-studio/dppn#1]
See if you can implement DPPN (crossover and mutation) with Keras, so that I don't have to implement gradient descent from scratch (Possibly by adding a layer with one output).

**Comment by jinyeom** [comment]
not going to happen; implementing from scratch
